src/driver/MongodbDriver.ts

```diff
@@ -126,6 +126,8 @@ export class MongodbDriver implements Driver {
                 cursor.sort(options.sort);
             if (options.snapshot)
                 cursor.snapshot(options.snapshot);
+            if (options.project)
```

Comment:
how about dynamically setting the value to cursor instead of a batch of if statement
something like this

```ts
            Object.getOwnPropertyNames(options).forEach(option => {
                if (typeof cursor[option] === 'function') {
                    cursor[option].call(cursor, <any>options[option]);
                }    
            });
```